src/main/java/Menu.java

```diff
@@ -50,11 +70,19 @@ private String setOption() {
 
     private boolean isInvalidOption(String option) {
         ArrayList<String> validIndexes = new ArrayList();
-
-        for (int index = 1; index <= Options.values().length; index++) {
+        for (int index = 1; index <= menuOptions.size(); index++) {
             validIndexes.add(Integer.toString(index));
         }
 
         return !validIndexes.contains(option);
     }
+
+    private String printPlayersMenu() {
+        String menu = "\nChoose an option:\n";
+        for (String printedMenuOption : printedMenuOptions) {
+            menu += printedMenuOption;
+        }
+
+        return gameUI.printMenuAndGetOption(menu);
```

Comment:
I am suspicious of methods that contain the word **And**. This is possibly due to the method not really doing just one thing only...